src/server/standalone.js

```diff
@@ -194,9 +195,19 @@ function StandAloneServer(gmeConfig) {
             }
         });
 
-        __httpServer.listen(gmeConfig.server.port, function () {
+        // something link
+        // https://github.com/rubenv/node-autoquit
+        // should be considered to shut the service down if it
+        // is not getting much use.
+
+        // https://github.com/herzi/systemd-socket
+        // https://nodejs.org/api/http.html#http_server_listen_handle_callback
+        __httpServer.listen(systemdSocket(0) || gmeConfig.server.port, function () {
```

Comment:
yeah, that's what I mean. I don't like the idea of introducing a systemd specific dependency...

This should be doable similar to how you can allow users to set the port w/ env variables: https://github.com/dfst/deepforge/blob/master/config/config.base.js#L10